### DISCUSSION TITLE
Option for use with self-signed certificates

### DISCUSSION THREAD

**Discussion by alansill** [darrenburns/posting#67] [Q&A]
I see we can configure a certificate path but for our use case there are a large number of private-network endpoints, each with their owned self-signed certificate. Is there an option like "curl -k" that can be used to bypass the certificate check on an https endpoint?

**Comment by darrenburns** [comment]
Are you looking for this? I believe `curl -k` and this option are equivalent.

<img width="593" alt="image" src="https://github.com/user-attachments/assets/bf730efe-e1c7-4cf7-8dc5-6abd66026382">
